content/en/docs/tasks/security/authorization/rbac-groups/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Authorization for groups and list claims
-description: Tutorial on how to configure the groups-base authorization and configure the authorization of list-typed claims in Istio.
+description: Tutorial on how to configure the groups-based authorization and configure the authorization of list-typed claims in Istio.
```

Comment:
Looks like this same typo appears in the opening line of the page (https://deploy-preview-6526--preliminary-istio.netlify.com/docs/tasks/security/authorization/rbac-groups/): "This tutorial walks you through examples to configure the groups-base authorization..." Do you want to fix that as well, while you're here?

Reply:
Fixed